src/tictactoe/test/tictactoe/JsonRepository/save_load_integration_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Tictactoe.JsonRepositoryTests do
+  use ExUnit.Case
+  alias Tictactoe.JsonRepository, as: SUT
+
+  describe "Json Repository save/load integration" do
+    test "saves map as json file and loads it from the file" do
+      timestamp = DateTime.utc_now()
+        |> DateTime.to_unix()
+      file_name = "integration-test-file-name-#{timestamp}"
```

Comment:
This is where it would be nice to be able to say

`repo = JsonRepository.init(".")` and then use it. At the end you could even
`JsonRepository.clean(repo)`